pandas/tests/plotting/common.py

```diff
@@ -173,6 +173,27 @@ def _check_visible(self, collections, visible=True):
         for patch in collections:
             assert patch.get_visible() == visible
 
+    def _check_filled(self, ax, filled=True):
+        """
+        Check each artist is filled or not
+
+        Parameters
+        ----------
+        ax : matplotlib Axes object or a numpy.ndarray of them
+        filled : bool
+            expected filling
+        """
+
+        def _check_patch_filling(artist):
+            for patch in artist.patches:
+                assert patch.fill == filled
+
+        if isinstance(ax, np.ndarray):
+            for rows in ax.flatten():
```

Comment:
i think there is an existing function called `_flatten_visible` in `common.py` which takes in both ax and list-like of ax, and return a flattened axes, could use that and do the conversion so that we don't need to have `if/else` here.

Reply:
Use `_flatten_visible` and removed if/else logic